config/foreman-proxy-content.migrations/200818160950-remove_tuning_fact.rb

```diff
@@ -0,0 +1,2 @@
+scenario[:facts] ||= {}
+scenario[:facts].delete('tuning') if scenario[:facts].key?('tuning')
```

Comment:
There's no need to introduce facts if you only want tor remove something.
```suggestion
scenario[:facts].delete('tuning') if scenario[:facts] && scenario[:facts].key?('tuning')
```